tests/affiliatedcertification/tests/affillated_certification_helm_chart.go

```diff
@@ -0,0 +1,103 @@
+			[]string{tsparams.TestPodLabel},
+			[]string{},
+			[]string{})
+		Expect(err).ToNot(HaveOccurred(), "error defining tnf config file")
+		err = namespaces.Create(tsparams.TestHelmChartCertified, globalhelper.GetAPIClient())
+		Expect(err).ToNot(HaveOccurred(), "Error creating namespace")
+	})
+
+	AfterEach(func() {
+		By("remove the project")
+		err := namespaces.Clean(tsparams.TestHelmChartCertified, globalhelper.GetAPIClient())
+		Expect(err).ToNot(HaveOccurred(), "Error delete ns affiliated-certification-helmchart-is-certified")
+	})
+
+	It("one helm to test,  are certified", func() {
+		By("check if helm is installed")
+		cmd := exec.Command("/bin/bash", "-c",
+			"helm vesrion")
+		err := cmd.Run()
+		if err != nil {
+			Skip("helm does not exist please install it to run the test.")
+		}
+
+		By("Install a helm chart")
+		cmd = exec.Command("/bin/bash", "-c",
+			"helm repo add openshift-helm-charts https://charts.openshift.io/ "+
+				"&& helm repo update && "+
+				"helm install example-vault1 openshift-helm-charts/hashicorp-vault -n "+tsparams.TestHelmChartCertified)
+		err = cmd.Run()
+		Expect(err).ToNot(HaveOccurred(), "Error installing helm chart")
+
+		By("Start test")
+		err = globalhelper.LaunchTests(
+			tsparams.TestHelmChartCertified,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).ToNot(HaveOccurred(), "Error running "+
+			tsparams.TestHelmChartCertified+" test")
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TestHelmChartCertified,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("one helm to test, chart not certified", func() {
+		By("check if helm is installed")
+		cmd := exec.Command("/bin/bash", "-c",
+			"helm vesrion")
+		err := cmd.Run()
+		if err != nil {
+			Skip("helm does not exist please install it to run the test.")
+		}
+
+		By("Create ns istio-system")
+		err = namespaces.Create("istio-system", globalhelper.GetAPIClient())
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Install a helm chart")
+		cmd = exec.Command("/bin/bash", "-c",
+			"helm repo add istio https://istio-release.storage.googleapis.com/charts "+
+				"&& helm repo update &&"+
+				"helm install istio-base istio/base --set defaultRevision=default -n "+tsparams.TestHelmChartCertified)
+		err = cmd.Run()
+		Expect(err).ToNot(HaveOccurred(), "Error installing helm chart")
+
+		By("Start test")
+		err = globalhelper.LaunchTests(
+			tsparams.TestHelmChartCertified,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).To(HaveOccurred(), "Error running "+
+			tsparams.TestHelmChartCertified+" test")
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TestHelmChartCertified,
+			globalparameters.TestCaseFailed)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("remove the istio-system")
```

Comment:
I would prefer here the helm way of deleting istio chart, not just deleting the istio namespace.

Reply:
resolved